source/server/admin/admin.cc

```diff
@@ -646,11 +653,9 @@ Http::ServerConnectionPtr AdminImpl::createCodec(Network::Connection& connection
 
 bool AdminImpl::createNetworkFilterChain(Network::Connection& connection,
                                          const std::vector<Network::FilterFactoryCb>&) {
-  // Don't pass in the overload manager so that the admin interface is accessible even when
```

Comment:
Move this comment to the NullOverloadManager to explain why we use it here rather than the real OverloadManager.